test/test_format.py

```diff
@@ -509,10 +537,10 @@ def test_nice_date(self):
                 i = i + 1
 
         # test fall back to english
-        dt = datetime.datetime(2018, 2, 4, 0, 2, 3)
-        self.assertEqual(nice_date(
-            dt, lang='invalid', now=datetime.datetime(2018, 2, 4, 0, 2, 3)),
-            'today')
+#        dt = datetime.datetime(2018, 2, 4, 0, 2, 3)
+#        self.assertEqual(nice_date(
+#            dt, lang='invalid', now=datetime.datetime(2018, 2, 4, 0, 2, 3)),
+#            'today')
```

Comment:
This should now be removed. A bad language code should now fail loudly. In this case, it would raise:

```
lingua_franca.internal.UnsupportedLanguageError: 
Language 'invalid' is not yet supported by Lingua Franca. Supported language codes include the following:
cs da de en es fr hu it nl pt sv